spec/models/enterprise_caching_spec.rb

```diff
@@ -66,23 +67,33 @@
         end
 
         context "with an order cycle" do
-          before { oc }
+          before do
+            oc
+            # We need to reload again after the OC touched the enterprise.
+            enterprise.reload
+          end
 
           it "touches enterprise when a classification on that product changes" do
+            pending "product touches distributor on change"
+
             expect {
               classification.save!
               enterprise.reload
```

Comment:
Ok, so if we don't reload on line 73 then the spec passes because the OC touches the enterprise. Do I understand the issue correctly?